astropy/utils/data.py

```diff
@@ -124,7 +124,8 @@ def get_readable_fileobj(name_or_obj, encoding=None, cache=False,
     manager that yields a readable file-like object.
 
     This supports passing filenames, URLs, and readable file-like
-    objects, any of which can be compressed in gzip or bzip2.
+    objects, any of which can be compressed in gzip, bzip2 or xz
+    (the latter on systems with lzma support).
```

Comment:
There's a number of situations where Python doesn't have bzip2 support (i.e. it's in the stdlib but not compiled).  Maybe  say "any of which can be compressed in gzip, bzip2 or xz if the appropriate compression libraries are available."